### PR TITLE
Release PR for 3.0

### PR DESCRIPTION
**Pull request by jfy133**
TODO:

- [x] Update versions
- [x] Update workflow diagram if necessary

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -11,6 +11,18 @@
 
 nextflow.enable.dsl = 2
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    IMPORT FUNCTIONS / MODULES / SUBWORKFLOWS / WORKFLOWS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+include { MAG  } from './workflows/mag'
```
Comment (maxulysse) [inline main.nf:20]: Harshil algniment?

File: modules.json [hunk 2]
```diff
@@ -210,9 +210,19 @@
             },
             "subworkflows": {
                 "nf-core": {
-                    "fasta_binning_concoct": {
```
Comment (maxulysse) [inline modules.json:213]: Was this subworkflow removed?
or was this overlooked by a TEMPLATE merge?

**Review by maxulysse — APPROVED** [review]
Minor comments

File: modules.json (hunk 2 above)
Comment (jfy133) [reply modules.json:213]: It shouldn't be!?

Comment (jfy133) [reply modules.json:213]: It's in the pipeline!

Comment (jfy133) [reply modules.json]: Had to remove and re-install... strange...

**Review by maxulysse — APPROVED** [review]
<3

**Review by CarsonJM — APPROVED** [review]
A monumental effort by @jfy133!!!

**Review by erikrikarddaniel — APPROVED** [review]
Looks good with a couple of questions:

1. Is it the new behaviour of `--input` that merits the 2.5.4 -> 3.0?
2. I saw a number of closed issues yesterday in emails, e.g. CheckM2 and dRep, but I don't see these changes included here.
3. Fonts in the svg files appear different

**Comment by jfy133** [comment]
> Looks good with a couple of questions:
> 
> 1. Is it the new behaviour of `--input` that merits the 2.5.4 -> 3.0?
> 2. I saw a number of closed issues yesterday in emails, e.g. CheckM2 and dRep, but I don't see these changes included here.
> 3. Fonts in the svg files appear different

Thanks @erikrikarddaniel !

To answer your questions: 

1. Yes exactly :) Because we drop support for direct fastq input `--input '*{R1,R2}.fasta.gz` , I consider this a backwards incompatible change, so following semvar this would equate to an API breaking change 
2. They've not been added to the pipeline, I just did some issue cleanup as I saw a bunch of duplicates, so just consolidating the issues
3. That's an system rendering issue thing on SVGs. I didn't convert text to paths, so if you don't have the font on your system it'll fallback to something else. As we normally only use PNGs for actual display, I leave the SVGs raw so the text remains editable:)

I hope that answers your questions!

**Comment by erikrikarddaniel** [comment]
Regarding no 2: Since I just started a run and saw the closed issues, I had some hopes of finding those features in 3.0, giving me extra motivation to look at the PR 😉.

**Review by prototaxites — APPROVED** [review]
One minor comment!

File: workflows/mag.nf [hunk 3]
```diff
@@ -664,8 +618,9 @@ workflow MAG {
         }
     } else {
         ch_assemblies_split = ch_input_assemblies
+            .dump(tag: 'ch_input_assemblies')
```
[the remark this answers is not in the document]
Comment (jfy133) [reply workflows/mag.nf:621]: Poop nope, good catch

Comment (jfy133) [inline workflows/mag.nf:621]:
```suggestion
```

**Pull request merged**